### PR TITLE
Multiple files in response & GI-plan

### PR DESCRIPTION
**Pull request by OlaStalberg**
- Database change: Payloadcontent is now byte[] instead of string. 
- Changes to frontend to support mutliple response files.
- Changes to backend to support multiple response files.
- Change to validation method for Json. Fixed bug where jsonvalidation didnt trigger error when jsonPath from the test was not found in response.
- Added new test GI-plan. Tests are set to supported: false as most of the tests dont include test-logic and gi-plan is removed as an option for protocol-selection in the frontend.

### CODE REVIEW

File: api/KS.FiksProtokollValidator.Tests/ValidatorTest.cs [hunk 1]
```diff
@@ -52,9 +52,9 @@ public void Setup()
             _fiksResponseKvittering = new FiksResponse
             {
                 Type = WebAPI.Resources.ResponseMessageTypes.KvitteringV1,
-                ReceivedAt = DateTime.Now,
+                ReceivedAt = DateTime.Now/*,
```
[the remark this answers is not in the document]
Comment (OlaStalberg) [reply api/KS.FiksProtokollValidator.Tests/ValidatorTest.cs:55]: Dette er for å bygget godkjent. Testen feiler, så jeg skal se om jeg kan rette på testen først :)

[the remark this answers is not in the document]
Comment (OlaStalberg) [reply api/KS.FiksProtokollValidator.Tests/ValidatorTest.cs]: Fikset i https://github.com/ks-no/fiks-protokoll-validator/pull/352/commits/35d21db21a39329b2985883fd7fcf9c18c54bb70

File: api/KS.FiksProtokollValidator.WebAPI/Resources/RequestMessageTypes.resx [hunk 2]
```diff
@@ -168,4 +168,37 @@
   <data name="OppdaterArealplanV2" xml:space="preserve">
     <value>no.ks.fiks.gi.plan.oppdatering.oppdaterarealplan.v2</value>
   </data>
+  <data name="FinnPlandokumenterV2" xml:space="preserve">
+    <value>no.ks.fiks.gi.plan.innsyn.finnplandokumenter.v2</value>
```
[the remark this answers is not in the document]
Comment (OlaStalberg) [reply api/KS.FiksProtokollValidator.WebAPI/Resources/RequestMessageTypes.resx:172]: Avventer endringene i punkt 1., 2. og 3. til vi flytter dette til et eget repo.

File: api/KS.FiksProtokollValidator.WebAPI/Resources/ResponseMessageTypes.resx [hunk 3]
```diff
@@ -159,4 +159,34 @@
...
+    <value>no.ks.fiks.gi.plan.innsyn.planfil.v2</value>
+  </data>
+  <data name="PlanområderV2" xml:space="preserve">
+    <value>no.ks.fiks.gi.plan.innsyn.planområder.v2</value>
+  </data>
+  <data name="RelatertePlanerV2" xml:space="preserve">
+    <value>no.ks.fiks.gi.plan.innsyn.relaterteplaner.v2</value>
+  </data>
+  <data name="PlandokumenterV2" xml:space="preserve">
+    <value>no.ks.fiks.gi.plan.innsyn.plandokumenter.v2</value>
```
[the remark this answers is not in the document]
Comment (OlaStalberg) [reply api/KS.FiksProtokollValidator.WebAPI/Resources/ResponseMessageTypes.resx:190]: Avventer endringene på dette til vi flytter dette til et eget repo.

File: web-ui/src/components/PayloadFile.vue [hunk 4]
```diff
@@ -75,12 +75,30 @@ export default {
...
-      const temporaryUrl = URL.createObjectURL(blob);
-      this.temporaryUrl = temporaryUrl; // Used to revoke URL
-      return temporaryUrl;
+      if (content.type == "application/octet-stream") {
+        const contentType = this.fileExtension === "pdf" ? { type: "application/pdf" } : null;
+        const blob = new Blob([content], contentType);
+        const temporaryUrl = URL.createObjectURL(blob);
+        this.temporaryUrl = temporaryUrl; // Used to revoke URL
+        return temporaryUrl;
+      }else{
```
[the remark this answers is not in the document]
Comment (OlaStalberg) [reply web-ui/src/components/PayloadFile.vue]: Fikset i commit https://github.com/ks-no/fiks-protokoll-validator/pull/352/commits/e956499e1c6a8682b2feeb9f3ca364dbb5e7edab

File: api/KS.FiksProtokollValidator.WebAPI/TestCases/no.ks.fiks.gi.plan.klient/TildelePlanidentN1/testInformation.json [hunk 5]
```diff
@@ -8,6 +8,6 @@
     "expectedResult": "Bekreftelse på mottat Opprettelse av arealplan, reur ny planident",
```
[the remark this answers is not in the document]
Comment (OlaStalberg) [reply api/KS.FiksProtokollValidator.WebAPI/TestCases/no.ks.fiks.gi.plan.klient/TildelePlanidentN1/testInformation.json]: Fikset i https://github.com/ks-no/fiks-protokoll-validator/pull/352/commits/1b1c96ca5f752f6c9ffbf46726983ca96ac2d582

File: web-ui/src/components/PayloadFile.vue [hunk 6]
```diff
@@ -75,12 +75,30 @@ export default {
...
+
+        let bytes = new Uint8Array(binaryLen);
+
+        for (let i = 0; i < binaryLen; i++) {
+            let ascii = decodedContent.charCodeAt(i);
+            bytes[i] = ascii;
+        }
+        
+        const mimeType = MimeTypes.lookup(this.fileExtension);
+        const contentType =  { type: mimeType };
```
[the remark this answers is not in the document]
Comment (OlaStalberg) [reply web-ui/src/components/PayloadFile.vue]: Fixed in https://github.com/ks-no/fiks-protokoll-validator/pull/352/commits/e9ebbe168a3c0af423d948068dfe86328d3ca634

**Review by jarleborsheim — APPROVED** [review]
LGTM 👍 
Husk regresjonstesting mot eksisterende protokoller

File: web-ui/src/components/PayloadFile.vue [hunk 7]
```diff
@@ -75,12 +75,30 @@ export default {
...
+      if (content.type == "application/octet-stream") {
+        const contentType = this.fileExtension === "pdf" ? { type: "application/pdf" } : null;
+        const blob = new Blob([content], contentType);
+        const temporaryUrl = URL.createObjectURL(blob);
+        this.temporaryUrl = temporaryUrl; // Used to revoke URL
+        return temporaryUrl;
+      }else{
+        const decodedContent = atob(content);
+        let binaryLen = decodedContent.length;
+
```
[the remark this answers is not in the document]
Comment (OlaStalberg) [reply web-ui/src/components/PayloadFile.vue:87]: Ja, men vi avventer dette

**Pull request merged**